src/common/config.cpp

```diff
@@ -24,11 +24,11 @@ bool isNeoMode() {
     return isNeo;
 }
 
-u32 getScreenWidth() {
+u32 GetScreenWidth() {
```

Comment:
Nit, the rest of the project uses camel case, it would be preferable to be consistent